packages/vscode-extension/src/debug/commonUtils.ts

```diff
@@ -109,36 +109,75 @@ export async function getDebugConfig(
   env?: string
 ): Promise<{ appId: string; env?: string } | undefined> {
   try {
-    if (isLocalSideloadingConfiguration) {
-      const localEnvManager = new LocalEnvManager(VsCodeLogInstance, ExtTelemetry.reporter);
-      const localSettings = await localEnvManager.getLocalSettings(ext.workspaceUri.fsPath);
-      return { appId: localSettings?.teamsApp?.teamsAppId as string };
+    if (isV3()) {
+      const inputs = getSystemInputs();
+      const getConfigRes = await core.getProjectConfigV3(inputs);
+      if (getConfigRes.isErr()) throw getConfigRes.error;
+      const config = getConfigRes.value;
+      if (!config)
+        throw new UserError("GetConfigError", "Failed to get project config", "Extension");
+      if (isLocalSideloadingConfiguration) {
```

Comment:
Does this mean `localSettings` is retired when `isV3` enabled?
Then which flag takes effect first, `isV3` or `isConfigUnifyEnabled`? @xiaolang124

Reply:
In this part, V3 has been forced to read local env. For now, we need to set ```isConfigUnifyEnabled``` to true to let create project generate local env config.

Reply:
When isV3 is enabled, I think there is no isConfigUnifyEnabled anymore.

Reply:
I still see `localSettings` in remaining code. Will that be retired by `isV3` or `isConfigUnifyEnabled`?

Reply:
unify config is the first step,  v3 is the second step.